frappe/core/doctype/server_script/server_script.py

```diff
@@ -188,6 +190,7 @@ def setup_scheduler_events(script_name, frequency):
 				"method": method,
 				"frequency": frequency,
 				"server_script": script_name,
+				"cron_format": cron_format,
```

Comment:
`cron_fromat` needs to be checked for existing scripts else updating cron value doesn't update the job :grimacing:

Reply:
check failing test